tests/k256_ecdh.cc

```diff
@@ -154,26 +153,39 @@ TEST_P(K256EcdhWycheproof, KAT)
   WycheproofEcdh test = GetParam();
 
   // Obtain public key from test.
-  vector<uint64_t> public_key(15);
+  bytes public_key(64);
   {
-    if (test.CompressedPoint) {
-      // Instead of parsing ASN.1, we just ignore the prefix ...
-      bytes prefix = from_hex("3056301006072a8648ce3d020106052b8104000a034200");
-      bytes compressed(test.public_key_asn1.begin() + prefix.size(),
-                       test.public_key_asn1.end());
-
-      bool res =
-        Hacl_EC_K256_point_decompress(compressed.data(), public_key.data());
+    // Instead of parsing ASN.1, we just ignore the prefix ...
+    if (test.InvalidAsn) {
```

Comment:
We should be able to do these too, but that's ok for now. Let's get this in when Victor is ok with the ocaml changes.